### ISSUE TITLE
Contradicting Test Cases 

### ISSUE THREAD

**Issue by SachinMore108** [LeetCode-Feedback/LeetCode-Feedback#868]
Question -  Longest Substring Without Repeating Characters

Test Case 1 - string passed "abcabcbb" 
Expected output : 3 ? How? it should be "abcabcb" (7)
Analysis: If we code accordingly to not repeat character in the substring than output 3 is correct

But same piece of code will fail in the second test case.

Test Case 2 - string passed "dvdf"
Expected output: 3? How? it should be 2 "dv" according to the analysis in Test case 1

**Comment by lc-ivan** [comment]
Hi @SachinMore108 ,
Thank you for reaching out to us. I've relayed this issue to our team to investigate.

**Comment by lc-ivan** [comment]
Hi @SachinMore108 ,
Thank you for your feedback regarding this problem. Upon review, our team has concluded that the problem and test cases are correct and will not be needing any further updates. In addition, we have added constraints to the problem. Though we have added constraints to the problem, this is a well-known issue for old problems, so we will not be rewarding LeetCoins for these cases. Thanks again and happy LeetCoding!
